index.html

```diff
@@ -383,7 +383,7 @@ <h3>MathML Element Mappings</h3>
 		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
 		    <span class="subrole">AXSubrole: <code>AXMathRoot</code></span><br />
 		    <span class="attrs">AXAttributes:
-		      <code>AXMathRootRadicand</code> pointing to the first in-flow child<br />
+		      <code>AXMathRootRadicand</code> is an array containing the first in-flow child as its unique element,<br />
 		      <code>AXMathRootIndex</code> pointing to the second in-flow child
```

Comment:
for the record: that part may look weird, but it's ok on the second glance: the original WebKit bug (https://bugs.webkit.org/show_bug.cgi?id=230986) has clarification that if a radicand is a single element, then the attribute returns an array consisting of one element, which makes sense.